helmfile/charts/grafana-ops/templates/backup-dashboard-configmap.yaml

```
@@ -0,0 +1,12 @@
+{{- if .Values.dashboards.backup.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: backupdashboard
+  namespace: monitoring
+  labels:
+    {{ .Values.labelKey }}: 'backup'
+data:
+  backup-dashboard.json: |-
+    {{- .Files.Get "dashboards/backup-dashboard.json" | nindent 4 }}
```

Maybe we could put multiple dashboards in one ConfigMap instead of one per dashboard?
I don't have a strong opinion on this, just wanted to mention it.

```suggestion
  backup-dashboard.json: |-
    {{- .Files.Get "dashboards/backup-dashboard.json" | nindent 4 }}
  {{- if .Values.dashboards.other.enabled }}
  other-dashboard.json: |-
    {{- .Files.Get "dashboards/other-dashboard.json" | nindent 4 }}
  {{- end }}
```

---

They are now in one configmap